.pullapprove.yml

```diff
@@ -0,0 +1,697 @@
+        files.include('/packages/core/src/render3/i18n.md') or
+        files.include('/packages/core/src/render3/interfaces/i18n.ts') or
+        files.include('/packages/common/locales/**') or
+        files.include('/packages/common/src/i18n/**') or
+        files.include('/packages/common/src/pipes/date_pipe.ts') or
+        files.include('/packages/common/src/pipes/i18n_plural_pipe.ts') or
+        files.include('/packages/common/src/pipes/i18n_select_pipe.ts') or
+        files.include('/packages/common/src/pipes/number_pipe.ts') or
+        files.include('/packages/compiler/src/i18n/**') or
+        files.include('/packages/compiler/src/render3/view/i18n/**') or
+        files.include('/packages/compiler-cli/src/extract_i18n.ts') or
+        files.include('/packages/localize/**') or
+        files.include('/aio/content/guide/i18n.md') or
+        files.include('/aio/content/examples/i18n/**')
+    reviewers:
+      teams:
+        - fw-i18n
+        - framework-global-approvers
+
+  # ================================================
+  #  Angular security
+  # ================================================
+  angular-security:
+    conditions:
+      - >
+        files.include('/packages/core/src/sanitization/**') or
+        files.include('/packages/core/test/linker/security_integration_spec.ts') or
+        files.include('/packages/compiler/src/schema/**') or
+        files.include('/packages/platform-browser/src/security/**') or
+        files.include('/aio/content/guide/security.md') or
+        files.include('/aio/content/examples/security/**') or
+        files.include('/aio/content/images/guide/security/**')
+    reviewers:
+      teams:
+        - fw-security
+        - framework-global-approvers
+
+  # ================================================
+  #  zone.js
+  # ================================================
+  zone-js:
+    conditions:
+      - >
+        files.include('/packages/zone.js/**')
+    reviewers:
+      teams:
+        - fw-zones
+        - framework-global-approvers
+
+  # ================================================
+  #  Benchpress
+  # ================================================
+  benchpress:
+    conditions:
+      - >
+        files.include('/packages/benchpress/**')
+    reviewers:
+      teams:
+        - tools-benchpress
+        - framework-global-approvers
+
+  # ================================================
+  #  Integration
+  # ================================================
+  integration:
+    conditions:
+      - >
+        files.include('/integration/**')
+    reviewers:
+      teams:
+        - fw-integration
+        - framework-global-approvers
+
+  # ================================================
+  #  Docs-infra
+  # ================================================
+  docs-infra:
+    conditions:
+      - >
+        files.include('/aio/*') or
+        files.include('/aio/aio-builds-setup/**') or
+        files.include('/aio/content/examples/*') or
```

Comment:
I'm guessing you migrated these over from CODEOWNERS or something similar? In pullapprove we use [fnmatch](https://docs.python.org/3/library/fnmatch.html) here and `**` will be the same as `*`. Will it cause any issues to use fnmatch instead? It's been on the list for a while to support glob too, but surprisingly nobody has actually needed it.

Reply:
Pretty sure you won't want the leading `/` either with fnmatch.